components/automate-ui/src/app/pages/policy/add-members/policy-add-members.component.ts

```diff
@@ -228,12 +251,16 @@ export class PolicyAddMembersComponent implements OnInit, OnDestroy {
   }
 
   public closeModal(): void {
+    // unsubscribe from Changes to the form
+    this.unsubscribeFromExpressionFormChanges();
```

Comment:
unsubscribing from the form when we no longer need it.

Reply:
The comment on L254 is not needed; same on L262. In fact, that falls under the, well, "very evil" category of comments. Please see https://www.red-gate.com/simple-talk/opinion/opinion-pieces/fighting-evil-code-comments-comments/ 😁